cmd/root.go

```diff
@@ -589,3 +589,19 @@ func login(ctx context.Context, cmd *cobra.Command, scopes []string) (context.Co
 
 	return ctx, oi, token, nil
 }
+
+func getAppUrl(frontend, app string) (string, error) {
+	if frontend == "" && app == "" {
+		return "", fmt.Errorf("'--app' flag must be set")
+	}
+	if frontend != "" && app != "" {
```

Comment:
```suggestion
	if frontend != "" && app != "" && frontend != app {
```

if the values are the same, we don't need to complain.